src/test/shared/sam/debugger/samDebugConfigProvider.test.ts

```diff
@@ -87,10 +87,11 @@ function assertEqualLaunchConfigs(actual: SamLaunchRequestArgs, expected: SamLau
     // Remove noisy properties before doing a deep-compare.
     for (const o of [actual, expected]) {
         delete o.manifestPath
-        delete o.documentUri
-        delete o.templatePath
-        delete o.workspaceFolder
-        delete o.codeRoot
+        // the next for are as any to prevent type issues (cannot delete required fields)
```

Comment:
looks like a typo or something. Could just omit the comment though, the `as any` is reasonably self-evident for typescript programmers.

Reply:
Reasonable, changed